windows/src/global/inc/keymankeyboard_id.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Name:             KeymanKeyboardIID
+ * Description:      Header file to include the keyman keyboard GUIDs
```

Comment:
@marc I am not sold on the name I have chosen for this header file `keymankeyboard_id.h`. I was also wondering if it would fit under an existing header such `keymanversion.h`

Reply:
I would call it kmtip_guids.h